server.js

```diff
@@ -6,6 +6,7 @@ const mongoose = require("mongoose");
 const session = require("express-session");
 
 //NOTE: ROUTER:
+const { setUserVarMiddleware } = require("./middlewares/auth_middleware");
```

Comment:
What is userVar?

Reply:
ah, I think :  userVar mean : global user variable. 
Should I update variable name to make more clear?